Libplanet/Blocks/BlockHeader.cs

```diff
@@ -1,3 +1,4 @@
+#nullable enable
```

Comment:
is it necessary?

Reply:
https://github.com/planetarium/libplanet/issues/458

Kinda. We are trying to migrate (albeit slowly) the existing codebase to have `null` types checked. 😅

Reply:
If you mean that it isn't strictly necessary since the `BlockHeader` class is strictly `struct` with all its properties also being a value type, then no, but I'd say it's still better to have it for management.